Assets/Scripts/SS3D/Systems/Health/BodyParts/HeadBodyPart.cs

```diff
@@ -81,14 +75,26 @@ protected override void BeforeDestroyingBodyPart()
             GetComponentInParent<Human>()?.DeactivateComponents();
         }
 
+        protected override void InflictDamage(BodyLayer layer, DamageTypeQuantity damageTypeQuantity)
+        {
```

Comment:
same here, the player should not die when the head is severed, only when the brain dies